frontend/lib/src/components/elements/Tabs/Tabs.tsx

```diff
@@ -117,7 +117,7 @@ function Tabs(props: TabProps): ReactElement {
           },
           TabBorder: {
             style: () => ({
-              backgroundColor: theme.colors.fadedText05,
+              backgroundColor: theme.colors.borderColorLight,
```

Comment:
sanity-check: is `borderColorLight` correct here or do we need a color that adapts to light/dark theme?

Reply:
Hmm, `borderColorLight` does adapt to light & dark theme correctly. It might not be the perfect name since for dark theme its actually darker than the normal border color. But the naming is anyway only temporary and internal as preparation for advanced theming. Do you have a better name suggestion or should leave it as `light` for now?

Reply:
ah ok, yeah makes sense. Maybe `borderColorLighter` or would make it more clear that it does not have anything to do with light and dark theme. But we can also leave it as is 👍

Reply:
Yeah, I think let's leave it as is and consider better naming for this later in the advanced theming project.